### ISSUE TITLE
How is shiny-monitor.R triggered?

### ISSUE THREAD

**Issue by trafficonese** [sjewo/shiny-loadbalancer#3]
Do you need to create a cronjob/service for `shiny-monitor.R`, or does it magically run if the app gets triggered?

Unfortunately the [original blog](withr.me/blog/2014/04/30/a-shiny-app-serves-as-shiny-server-load-balancer/) post does't exist anymore and redirects to another webpage.

**Comment by sjewo** [comment]
To be honest, i can't remember in detail how to use the script. The scripts just checks how many connections a R-session holds. It will be triggered by opening the Shiny-App defined in "server.R", which will redirect to one of multiple copys of a shiny-app (appdir_1, appdir_2, and so on). 

You can find a blog post about the general idea on r-bloggers: https://www.r-bloggers.com/some-thoughts-on-shiny-open-source-internal-load-balancing/

**Comment by trafficonese** [comment]
I know that blog and some others that all provide a similar idea, but none of them provide full information and some even have "wrong" or old information.

I got it running, but I had to adapt quite a bit and I triggered the monitor script manually, but I think it has to be installed as cronjob, a the blogpost also states.

I'll play around a bit more and add some information in the Readme. 
If you want I'll put it together in a PR.
